recipes/glslang/all/conanfile.py

```diff
@@ -40,9 +42,12 @@ class GlslangConan(ConanFile):
 
     short_paths = True
 
+    @property
+    def _is_v1(self):
+        return Version(self.version).major == 1
+
```

Comment:
If you're removing the older versions, is this check necessary? Isn't it always major version 1 now?